integration-tests/ahm/src/tests.rs

```diff
@@ -297,54 +301,165 @@ fn sovereign_account_translation() {
 	}
 }
 
-/// For human consumption.
+/// This test updates the `pallets/ah-migrator/src/sovereign_account_translation.rs` file.
+///
+/// It iterates through all possible Para-IDs (alive or dead) and their sovereign accounts and their
+/// first 100 derived accounts. If an account is found, it is added to the translation map. The
+/// value of 100 is arbitrary but nobody seems to use more than 10 in practice. The theoretical
+/// limit is 2^16, but that would make the test ~655 times slower.
+#[ignore]
 #[tokio::test]
-async fn print_sovereign_account_translation() {
-	let (mut rc, mut ah) = load_externalities().await.unwrap();
+async fn find_translatable_accounts() {
+	let mut rc = remote_ext_test_setup(Chain::Relay).await.unwrap();
 
-	let mut rc_to_ah = BTreeMap::new();
+	// Extract all accounts from the RC
+	let rc_accounts =
+		rc.execute_with(|| frame_system::Account::<Polkadot>::iter_keys().collect::<BTreeSet<_>>());
+	println!("Found {} RC accounts", rc_accounts.len());
+
+	// Para ID -> (RC sovereign, AH sovereign)
+	let mut sov_translations = BTreeMap::<u32, (AccountId32, AccountId32)>::new();
+	// Para ID -> (RC derived, index, AH derived)
+	let mut derived_translations = BTreeMap::<u32, (AccountId32, u16, AccountId32)>::new();
+
+	// Try to find Para sovereign and derived accounts.
+	for para_id in 0..(u16::MAX as u32) {
+		// The Parachain sovereign account ID on the relay chain
+		let rc_para_sov =
+			xcm_builder::ChildParachainConvertsVia::<ParaId, AccountId32>::convert_location(
+				&Location::new(0, Junction::Parachain(para_id.into())),
+			)
+			.unwrap();
 
-	rc.execute_with(|| {
-		for para_id in paras_registrar::Paras::<Polkadot>::iter_keys().collect::<Vec<_>>() {
-			let rc_acc =
-				xcm_builder::ChildParachainConvertsVia::<ParaId, AccountId32>::convert_location(
-					&Location::new(0, Junction::Parachain(para_id.into())),
-				)
+		let (ah_para_sibl, found_para_id) =
```

Comment:
can be also sanity checked with `SiblingParachainConvertsVia<Sibling, AccountId>` result

Reply:
Hm this seems to produce the wrong account ID actually, it is the same as the child one, see: https://github.com/paritytech/polkadot-sdk/blob/1dbaeba9044ef491847a8204da98cd9e74534cdb/polkadot/xcm/xcm-builder/src/location_conversion.rs#L302-L314 

Will ask in XCM chat, but for now it seems wrong.